docs/src/main/sphinx/security/security-overview.rst

```diff
@@ -0,0 +1,159 @@
+   <https>`, formerly known as SSL.
+
+The Trino server uses the standard HTTPS protocol and TLS encryption, formerly
+knownn as SSL.
+
+.. _cl-access-auth:
+
+Authentication
+^^^^^^^^^^^^^^
+
+Trino supports several authentication providers. When setting up a new cluster,
+start with simple password file authentication before configuring another
+provider.
+
+* :doc:`Password file authentication <password-file>`
+* :doc:`LDAP authentication <ldap>`
+* :doc:`OAuth 2.0 authentication <oauth2>`
+* :doc:`Salesforce authentication <salesforce>`
+
+.. _user-name-management:
+
+User name management
+""""""""""""""""""""
+
+Trino provides ways to map the user and group names from authentication
+providers to Trino user names.
+
+* :doc:`User mapping <user-mapping>` applies to all authentication systems,
+  and allows for JSON files to specify rules to map complex user names from
+  some systems (``alice@example.com``) to simple user names (``alice``).
+* :doc:`File-based group provider <group-file>` provides a way to assign a set
+  of user names to a group name to ease access control.
+
+.. _cl-access-control:
+
+Authorization and access control
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. comment:
+   Trino and the included enhanced connectors allow you to :doc:`control
+   access <access-control>` to the data exposed by Trino in configured data
+   sources.
+
+Trino's :doc:`default method of access control <built-in-system-access-control>`
+allows all operations for all authenticated users.
+
+To implement access control, use:
+
+* :doc:`File-based system access control <file-system-access-control>`, where
+  you configure JSON files that specify fine-grained user access restrictions at
+  the catalog, schema, or table level.
+* Configure :doc:`Kerboeros authentication <server>` over HTTPS for client
+  access to the coordinator.
+* Configure the Trino :doc:`CLI <cli>` for connecting to a :doc:`Kerberos
+  enabled </security/cli>` coordinator.
+
+* In addition, Trino :doc:`provides an API </develop/system-access-control>`
+  that allows you to create a custom access control method, or to extend an
+  existing one.
+
+.. _security-inside-cluster:
+
+Securing inside the cluster
+---------------------------
+
+You can :doc:`secure the internal communication <internal-communication>`
+between coordinator and workers inside the clusters.
+
+Secrets in properties files, such as passwords in catalog files, can be secured
+with :doc:`secrets management <secrets>`.
+
+.. _security-data-sources:
+
+Securing cluster access to data sources
+---------------------------------------
+
+Communication between the Trino cluster and data sources is configured for each
+catalog. Each catalog uses a connector, which supports a variety of
+security-related configurations.
+
+More information is available with the documentation for individual
+:doc:`connectors </connector>`.
+
+:doc:`Secrets management <secrets>` can be used for the catalog properties files
```

Comment:
This kind of duplicates the same stuff above. Not sure we really need to .. but maybe its good to do.. wdyt?

Reply:
The division between these sections makes more sense upstream.